crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -496,6 +496,19 @@ bitflags::bitflags! {
         const DEPTH_CLAMP_ORTHO                 = (1 << 10);
         const TEMPORAL_JITTER                   = (1 << 11);
         const MORPH_TARGETS                     = (1 << 12);
+        /// Whether the view has motion vectors.
+        ///
+        /// This is distinct from [`MeshPipelineKey::MOTION_VECTOR_PREPASS`] as the key
+        /// represents whether the specific mesh being rendered supports motion vectors,
+        /// while the `VIEW_MOTION_VECTOR_PREPASS` represents whether the whole view should support
+        /// motion vectors.
+        ///
+        /// This is an important distinction. There may be several meshes drawn into a
+        /// buffer, including the motion vector buffer.
+        ///
+        /// Used in the prepass pipeline.
+        const VIEW_MOTION_VECTOR_PREPASS        = (1 << 13);
```

Comment:
Do we want / need this anymore? I thought the changes to `MOTION_VECTOR_PREPASS` were meant to indicate when a motion vector prepass is being/has been run and so motion vectors are present.

Reply:
There is still a distinction between "Does this view have a motion vector buffer" and "does this mesh support motion vectors" for the specific mesh being rendered.